pysat/model_utils.py

```diff
@@ -304,13 +305,16 @@ def collect_inst_model_pairs(start=None, stop=None, tinc=None, inst=None,
                 inst.load(date=istart)
 
             if not inst.empty and inst.index[0] >= istart:
-                added_names = extract_modelled_observations(inst=inst, \
-                                model=mdata, inst_name=inst_name,
-                                                            mod_name=mod_name, \
-                                mod_datetime_name=mod_datetime_name, \
-                                mod_time_name=mod_time_name, \
-                                mod_units=mod_units, sel_name=sel_name, \
-                                method=method, model_label=model_label)
+                added_names = extract_modelled_observations(inst=inst,
```

Comment:
Doesn't this cause the lines to be longer than 80 characters?

Reply:
to preserve readability, I agree.  Changing back